app/components/verkiezingen/burgemeester-alert.js

```diff
@@ -1,47 +1,31 @@
-import { service } from '@ember/service';
 import Component from '@glimmer/component';
+
+import { action } from '@ember/object';
+import { service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
-import { task, timeout } from 'ember-concurrency';
+
+import { restartableTask, timeout } from 'ember-concurrency';
+import { consume } from 'ember-provide-consume-context';
+
 import {
   MANDAAT_AANGEWEZEN_BURGEMEESTER_CODE,
   MANDAAT_BURGEMEESTER_CODE,
 } from 'frontend-lmb/utils/well-known-uris';
+import { INPUT_DEBOUNCE } from 'frontend-lmb/utils/constants';
 
-export default class VerkiezingenBcsdVoorzitterAlertComponent extends Component {
+export default class VerkiezingenBurgemeesterAlertComponent extends Component {
+  @consume('alert-group') alerts;
   @service store;
-  @service installatievergadering;
 
-  @tracked errorMessage = '';
-  @tracked lastRecomputeTime = null;
-
-  constructor() {
-    super(...arguments);
-    this.installatievergadering.forceRecomputeBCSD();
-  }
+  @tracked errorMessageId = 'cb8e18dd-647a-452b-a2a3-67bb644cfc4e';
+  @tracked errorMessage;
 
   get collegeOrgaanInTijd() {
     return this.args.collegeBestuursorgaanInTijd;
   }
 
-  // Polling because otherwise changes in the installatievergadering data are not picked up
-  handleErrorMessage = task({ restartable: true }, async () => {
-    if (this.isDestroyed) {
-      return;
-    }
-    if (
-      this.lastRecomputeTime &&
-      this.lastRecomputeTime ===
-        this.installatievergadering.recomputeBCSDNeededTime
-    ) {
-      await timeout(5000);
-      // nothing for now let's try again later
-      this.handleErrorMessage.perform();
-      return;
-    }
-
-    this.lastRecomputeTime =
-      this.installatievergadering.recomputeBCSDNeededTime;
-
+  handleErrorMessage = restartableTask(async () => {
+    await timeout(INPUT_DEBOUNCE);
```

Comment:
should also be removed

Reply:
Sometimes these methods are called quickly after each other because everything start calling the forceComputeBcsd, this prevents that the method is only executed when this triggers are done after the debounce. Not sure this is a good way, any thoughts?